firebase-vscode/src/html-scaffold.ts

```diff
@@ -0,0 +1,67 @@
+import { Uri, Webview } from "vscode";
+
+export function getHtmlForWebview(
+  entryName: string,
+  extensionUri: Uri,
+  webview: Webview
+) {
+  const scriptUri = webview.asWebviewUri(
+    Uri.joinPath(extensionUri, `dist/web-${entryName}.js`)
+  );
+  const styleUri = webview.asWebviewUri(
+    Uri.joinPath(extensionUri, `dist/web-${entryName}.css`)
+  );
+  const moniconWoffUri = webview.asWebviewUri(
+    Uri.joinPath(extensionUri, "resources/Monicons.woff")
+  );
+  const codiconsUri = webview.asWebviewUri(
+    Uri.joinPath(extensionUri, "resources/dist/codicon.css")
+  );
+  // Use a nonce to only allow a specific script to be run.
+  const nonce = getNonce();
+
+  return `<!DOCTYPE html>
+			<html lang="en">
```

Comment:
Nit: tabs -> spaces and formatting to fit. If we start on the next line we can get back ~10 characters or so

Reply:
Done. I think.